app/views/bastion/layouts/assets.html.erb

```diff
@@ -20,6 +20,7 @@
     angular.module('Bastion').value('CurrentOrganization', "<%= Organization.current.id if Organization.current %>");
     angular.module('Bastion').value('contentAccessMode', "<%= Organization.current.try(:content_access_mode) if Organization.current %>");
     angular.module('Bastion').value('foreman', tfm);
+    angular.module('Bastion').value('repositoryTypes', angular.fromJson('<%= Katello::RepositoryTypeManager.repository_types.values.to_json.html_safe %>'));
```

Comment:
This change makes it seem even more odd that bastion and katello are separate as it sets a new precedent by introducing katello concepts into bastion.  Really make me want to combine bastion and katello some day.

Reply:
we've already done this with katello specific settings.

Reply:
such as unregister_delete_host

Reply:
True, I'm not saying I'm against the change but that I'm in favor of merging bastion and katello.